hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -406,7 +406,13 @@ def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None
 
         vehicle.engine_is_running = get_child_value(state, "DrivingReady")
 
-        # TODO: vehicle.air_temperature = get_child_value(state, "Cabin.HVAC.Driver.Temperature.Value")
+        air_temp = get_child_value(
+            state,
+            "Cabin.HVAC.Row1.Driver.Temperature.Value",
```

Comment:
Looks like no unit set now.

Reply:
This is just the retrieval into a local variable, it is set on line 415

Reply:
Line 415 looks to be adjusting for off.